### DISCUSSION TITLE
If this an add-on extension for SD?

### DISCUSSION THREAD

**Discussion by wktra** [parlance-zz/g-diffuser-bot#85] [Q&A] [answered]
If this an add-on extension for SD?
How do I install it? 

Or is this only for discord use?

**Comment by parlance-zz** [comment, marked as answer]
This is an add-on / extension of Stable Diffusion.

In addition to the Discord bot there is also a purely local "Interactive CLI". It works similarly to the discord bot but is just a command line window. See the installation and running instructions here: https://github.com/parlance-zz/g-diffuser-bot/blob/dev/README.md#Installation
